search/tutorials/retail_env_setup.md

```diff
@@ -0,0 +1,120 @@
+# Getting started with Google Cloud Retail (Python)
+
+<walkthrough-disable-features toc></walkthrough-disable-features>
+
+#
+
+
+Learn how to call the Google Cloud Retail API using client libraries:
+
+1.  Enable the Retail API in a Google Cloud project.
+
+2.  Set up authentication.
+
+3.  Use code samples to call the Retail API.
+
+Estimated time to complete:
+<walkthrough-tutorial-duration duration="3"></walkthrough-tutorial-duration>
+
+To get started, click **Start**.
+
+## Select your project
+
+Google Cloud Platform organizes resources into projects. This allows you to
+collect all the related resources for a single application in one place.
+
+<walkthrough-project-setup billing="true"></walkthrough-project-setup>
+
+To learn how to enable the Cloud Translation API, click **Next**.
+
+## Enable the Retail API
+
+Enable the Retail API by clicking **Enable APIs**:
+
+<walkthrough-enable-apis apis="retail.googleapis.com">
+    </walkthrough-enable-apis>
+
+To learn how to set up your application, click **Next**.
+
+
+## Set up authentication
+
+To run a code sample from the Cloud Shell, you need to authenticate. Luckily, Google Cloud makes it easy to use the Application Default Credentials. 
+
+Open Terminal and run the following command to set the project:
+```bash
+gcloud config set project <YOUR_PROJECT_ID>
+```
+
+Click **Authorize** in the Authorization popup.
+
+Next, set your user credentials to authenticate your requests to Retail API
+
+```bash
+gcloud auth application-default login
+```
+
+Type 'Y' and press Enter. Click on the link in Terminal. A browser window should open asking you to login using your gmail account.
```

Comment:
Type 'Y' and press Enter. Click on the link in the Terminal. A browser window should appear asking you to log in using your gmail account.